website/docs/tools/analyzer.md

```diff
@@ -13,6 +13,9 @@ It can be further processed or manually edited before passing it to one of the o
 Currently, the following package managers (grouped by the programming language they are most commonly used with) are supported:
 
 * C / C++
+  * [Bazel](https://bazel.build/) (**experimental**) (limitations:
+    bzlmod only, *very* basic parsing of `MODULE.bazel` files, patches to source artifacts are not included,
```

Comment:
Could we re-word the `bzlmod only` item a bit. I do not (as a person not into bazel) understand which limitation it refers to, e.g. what is missing? 

Also for the second item, could we make the text indicate what is missing? (basic parsing is not a limitation in case all needed data is available).

Reply:
I've reworded the text and mentioned the limited testing.

Reply:
Thanks!